internal/model/sharedpullerstate.go

```diff
@@ -96,7 +96,7 @@ func (s *sharedPullerState) tempFile() (io.WriterAt, error) {
 		return nil, err
 	} else if info.Mode()&0200 == 0 {
 		err := os.Chmod(dir, 0755)
-		if err == nil {
+		if !ignorePerms && err == nil {
```

Comment:
Expanding on Audrius' question - does this really work? :) As I understand it, we only do the chmod if the user doesn't have write permissions on the directory. If they don't, then we *need* to do the chmod, or the subsequent create will false, ignorePerms or no? Or is the issue that the directory has a weird permission mask so that we have write access but fail the test (i.e. `dr-xr-xrwx` or something)?